relayer/relays/parachain/beefy-listener.go

```diff
@@ -82,125 +85,65 @@ func (li *BeefyListener) Start(ctx context.Context, eg *errgroup.Group) error {
 	eg.Go(func() error {
 		defer close(li.tasks)
 
-		// Subscribe NewMMRRoot event logs and fetch parachain message commitments
-		// since latest beefy block
-		beefyBlockNumber, _, err := li.fetchLatestBeefyBlock(ctx)
-		if err != nil {
-			return fmt.Errorf("fetch latest beefy block: %w", err)
-		}
-
-		err = li.doScan(ctx, beefyBlockNumber)
-		if err != nil {
-			return fmt.Errorf("scan for sync tasks bounded by BEEFY block %v: %w", beefyBlockNumber, err)
-		}
+		// Add some randomness here in case one relayer is down and other relayers won't compete for
+		// that failed message at same time.
+		ticker := time.NewTicker(time.Second*30 + time.Duration(rand.Intn(10))*time.Second)
+		for {
+			beefyBlockNumber, _, err := li.fetchLatestBeefyBlock(ctx)
+			if err != nil {
+				return fmt.Errorf("fetch latest beefy block: %w", err)
+			}
 
-		err = li.subscribeNewMMRRoots(ctx)
```

Comment:
`subscribeNewMMRRoots` will get trigged when there is new MMRRoot bridged(i.e. when session change which is 4 hours for Polkadot network). 

When switching to a multiple relayer context with task scheduled one message per relayer in a round-robin manner.  In case when a relayer is down, `subscribeNewMMRRoots` does not make sense for avoid all relayers competing for a single message at the same time.

So a periodical scan with some randomness make more sense, each round a single message is processed by a relayer, submit anyway when that message is timeout without anyone picked up.